Makefile

```diff
@@ -371,7 +371,7 @@ install-tools:
 		golang.org/x/lint/golint \
 		golang.org/x/tools/cmd/goimports \
 		github.com/securego/gosec/cmd/gosec@v0.0.0-20191008095658-28c1128b7336 \
-		sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.4 \
+		sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0 \
```

Comment:
This doesn't belong to this PR, I believe.

Reply:
Yes, it should belong to this [PR](https://github.com/jaegertracing/jaeger-operator/pull/1305).
![image](https://user-images.githubusercontent.com/3659110/105751404-ec4f3a80-5f80-11eb-897b-391e121157b0.png)
I just don't want to open another pr for this small fix.

Reply:
Alright, that's fine.